.github/workflows/e2e.yaml

```diff
@@ -92,13 +92,14 @@ jobs:
           terraform plan \
```

Comment:
@mxsrc we have also reference the new aws path here. we are currently still making use of the base directory